docs: document /tracker/events changes

teleivo opened the pull request.

Bekkalizer reviewed (commented):
Hey @teleivo ! Looks good! I updated the release note document with some of the capture app features, to show the format. Perhaps you can reformat the PR in the same way as the examples I now added to the master branch for 2.39 release notes?

teleivo commented:
> Hey @teleivo ! Looks good! I updated the release note document with some of the capture app features, to show the format. Perhaps you can reformat the PR in the same way as the examples I now added to the master branch for 2.39 release notes?

@Bekkalizer only saw this now 😬 2.39 is already out, so I assume its release notes as well. Is this PR still needed, otherwise you/I can close it.

teleivo commented:
Closing as 2.39 has been released already. @Bekkalizer feel free to reopen/reuse the content of the PR.
